practice/lib/regex/functionality/regex_patterns.rb

```diff
@@ -0,0 +1,27 @@
+module RegexPatterns
+  # Path to the file directory, file's name is given as argument in methods below
+  FILE_PATH = File.join(Dir.home, 'RubyTasks', 'practice', 'lib', 'regex', 'test_data').freeze
+
+  # Method should return true if there are zero occurrences of the string in the text data and false otherwise
+  def self.zero_occurrence?(file_name, text_pattern)
```

Comment:
I'm not sure if we want to have a lot of separate methods for very specific cases as 0, 1, 10 occurrences etc.
I believe, it's better to have 1 method which returns number of matches. The number that you expect should be checked in a test.